simplivity/resources/hosts.py

```diff
@@ -151,3 +151,22 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+        self._hosts = Hosts(self._connection)
+
+    def remove_from_federation(self, force=False, timeout=-1):
```

Comment:
Can we rename this method as remove/delete to make the method name short and aligned with other resources delete methods?

Reply:
I renamed the method to `remove`